examples/user_guide/Geometry_Data.ipynb

```diff
@@ -19,7 +19,7 @@
     "import holoviews as hv\n",
     "from holoviews import opts\n",
     "\n",
-    "hv.extension('bokeh')\n",
+    "hv.extension('matplotlib')\n",
```

Comment:
Why the change in the loaded plotting extension?

Reply:
Accidental.